lapis2-docs/src/components/Configuration/MetadataTypesList.astro

```diff
@@ -0,0 +1,30 @@
+---
+import { OnlyIf } from '../OnlyIf.tsx';
+import { getConfig } from '../../config.ts';
+---
+
+SILO currently supports the following metadata types:
+
+<ul>
+    <li><code>string</code></li>
+    <li><code>int</code></li>
+    <li><code>float</code></li>
+    <li>
+        <code>pango_lineage</code>: Systematic classification of lineage with inheritance structure that can be computed
+        for some pathogens.
+        <OnlyIf condition={!!getConfig().schema.metadata.find((m) => m.type === 'pango_lineage')}>
+            Also see <a href='/concepts/pango-lineage-query'>here</a>.
+        </OnlyIf>
+    </li>
+    <li><code>date</code>: Values must be valid dates in the form <code>YYYY-MM-DD</code>.</li>
+    <li>
+        <code>insertion</code>: A comma separated list of nucleotide insertions. Each insertion has the form
+        <code>{'<segment>:<position>:<symbols>'}</code>. Example value:
```

Comment:
No, the next sentence explains that. I think - since this is a reference for maintainers that setup an instance - it's better to tell them the details instead of filtering by available config.